Assets/MRTK/Core/Utilities/Editor/SizeUtilities.cs

```diff
@@ -34,10 +34,22 @@ public static void RendererSize()
             }
         }
 
+        [MenuItem("GameObject/MRTK Debug Utilities/Print Renderer Size", true, 41)]
```

Comment:
It looks like this `MenuItem` has the same name as line 17 (same with the other two `MenuItem`s). Should this one be something like 
```suggestion
        [MenuItem("GameObject/MRTK Debug Utilities/Validate Renderer", true, 41)]
```
though, it also doesn't look like this prints anything?

Reply:
it's intentional that it has the same menu item - the validation method will just make sure that menu item is grayed out - (you flag a validation method by the bool in the menuitem attribute)

Reply:
ahhh but now i just realize that this one has the wrong priority - which didn't seem to bother unity for some reason - but i'll patch it up anyways :D , should be 40, not 41

Reply:
Oh cool! I didn't realize that was a thing, awesome!